src/client/app/components/meters/CreateMeterModalComponent.tsx

```diff
@@ -428,7 +433,9 @@ export default function CreateMeterModalComponent() {
 							<Input id='cumulativeResetStart' name='cumulativeResetStart' type='text' autoComplete='off'
 								onChange={e => handleStringChange(e)}
 								value={meterDetails.cumulativeResetStart}
-								placeholder='HH:MM:SS' />
+								placeholder='HH:MM:SS'
+								disabled={meterDetails.cumulativeReset === false || meterDetails.cumulative === false ? true : false}
```

Comment:
A small point but can
```meterDetails.cumulative === false ? true : false```
become
```meterDetails.cumulative === false```
This also mirrors the cumulativeReset check.

This also applies below and in the two similar cases for edit.

Reply:
done.